eth/contracts/PancakeStableSwap/PancakeStableSwapTwoPool.sol

```diff
@@ -0,0 +1,774 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import "@openzeppelin-4.5.0/contracts/access/Ownable.sol";
+import "@openzeppelin-4.5.0/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin-4.5.0/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "@openzeppelin-4.5.0/contracts/security/ReentrancyGuard.sol";
+import "./IPancakeStableSwapLP.sol";
+
+contract PancakeStableSwapTwoPool is Ownable, ReentrancyGuard {
+    using SafeERC20 for IERC20;
+
+    uint256 public constant N_COINS = 2;
+
+    uint256 public constant MAX_DECIMAL = 18;
+    uint256 public constant FEE_DENOMINATOR = 1e10;
+    uint256 public constant PRECISION = 1e18;
+    uint256[N_COINS] public PRECISION_MUL;
+    uint256[N_COINS] public RATES;
+
+    uint256 public constant MAX_ADMIN_FEE = 1e10;
+    uint256 public constant MAX_FEE = 5e9;
+    uint256 public constant MAX_A = 1e6;
+    uint256 public constant MAX_A_CHANGE = 10;
+    uint256 public constant MIN_BNB_GAS = 2300;
+    uint256 public constant MAX_BNB_GAS = 23000;
```

Comment:
I think we want to replace `BNB` by a neutral name for a gas token, like `NATIVE`. Otherwise it will be confusing.

Reply:
Sure, but later :P